api-tests/src/test/java/org/openmrs/module/reporting/cohort/definition/evaluator/DrugOrderCohortDefinitionEvaluatorTest.java

```diff
@@ -0,0 +1,109 @@
+/*
+ * The contents of this file are subject to the OpenMRS Public License
+ * Version 1.0 (the "License"); you may not use this file except in
+ * compliance with the License. You may obtain a copy of the License at
+ * http://license.openmrs.org
+ *
+ * Software distributed under the License is distributed on an "AS IS"
+ * basis, WITHOUT WARRANTY OF ANY KIND, either express or implied. See the
+ * License for the specific language governing rights and limitations
+ * under the License.
+ *
+ * Copyright (C) OpenMRS, LLC.  All Rights Reserved.
+ */
+
+package org.openmrs.module.reporting.cohort.definition.evaluator;
+
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.openmrs.Cohort;
+import org.openmrs.api.context.Context;
+import org.openmrs.module.reporting.cohort.definition.DrugOrderCohortDefinition;
+import org.openmrs.module.reporting.cohort.definition.service.CohortDefinitionService;
+import org.openmrs.test.BaseModuleContextSensitiveTest;
+
+public class DrugOrderCohortDefinitionEvaluatorTest extends BaseModuleContextSensitiveTest {
+  protected static final String TEST_DATA = "org/openmrs/module/reporting/include/ReportTestDataset-openmrs-1.10.xml";
+  private DrugOrderCohortDefinition cohortDefinition;
+
+  @Before
+  public void setup() throws Exception {
+    cohortDefinition = new DrugOrderCohortDefinition();
+    executeDataSet(TEST_DATA);
+  }
+
+  @After
+  public void tearDown() {
+    cohortDefinition = null;
+  }
+
+  @Test
+  public void shouldReturnAllPatientsTakingAnyofListedDrugs() throws Exception {
+    cohortDefinition.setOnlyCurrentlyActive(Boolean.FALSE);
+    Cohort cohort = Context.getService(CohortDefinitionService.class).evaluate(cohortDefinition, null);
+    System.out.println("Print out " + cohort.getMemberIds() + " " + cohortDefinition.getOnlyCurrentlyActive());
+
+  }
+
+  // @Test
+  // public void shouldReturnAllPatientsNotTakingAnyOfListedDrugs() throws Exception {
```

Comment:
What are these commented out methods for?

Reply:
had just commented it out bse I was planning to implement it later